pkg/file/scripts/install_rpm.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+dnf install --assumeyes "$INSTALLER_PATH"
```

Comment:
Do we want to use dnf here rather than either `yum` or `rpm -i`? Older Red Hat versions (<= 7) don't have DNF AFAIK.

Same comments for uninstall scripts/golden.

Reply:
Good question.

I believe we should support newest/current-default package manager.
Users can customize the (un)install scripts for older versions of Fedora/RHEL.

> DNF was introduced in [Fedora 18](https://en.wikipedia.org/wiki/Fedora_version_history#Fedora_18) in 2013;[[11]](https://en.wikipedia.org/wiki/DNF_(software)#cite_note-DNF-Fedora-11) it has been the default package manager since [Fedora 22](https://en.wikipedia.org/wiki/Fedora_version_history#Fedora_22) in 2015,[[12]](https://en.wikipedia.org/wiki/DNF_(software)#cite_note-F22_released-12) [Red Hat Enterprise Linux](https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux) 8,

I'll defer to the product team, @noahtalerman @rachaelshaw

Reply:
Just hopped onto design review and, given the target use case, we can stick with dnf.

We probably want to add a note on [this doc](https://fleetdm.com/docs/get-started/faq#what-host-operating-systems-does-fleet-support) explaining that we're assuming dnf on our software install scripts, so hosts on CentOS 7 will either need to have dnf installed or will need revised (un)install scripts, but this doesn't block the code path you've taken.

Reply:
I guess I'm actually on docs for RPM stuff, so lemme get a PR in for this rq.